Framework/DataHandling/src/ProcessBankData.cpp

```diff
@@ -247,7 +250,8 @@ void ProcessBankData::run() {
   }
 
 #ifndef _WIN32
-  alg->getLogger().debug() << "Time to process " << entry_name << " " << m_timer << "\n";
+  if (alg->getLogger().isDebug())
+    alg->getLogger().debug() << "Time to ProcessBankData " << entry_name << " " << timer << "\n";
```

Comment:
Just trying to understand the reason for needing `isDebug` - does sending data to the logger have unintended consequences like a slow down even when the logger is not in debug mode?

Reply:
Unfortunately, it can. While the logger will swallow the string, the string is still generated. It would take some time to track the PR down that had it, but I deleted a debug log inside FitPeaks and got a couple of seconds faster execution. This probably doesn't need it because there shouldn't be a ton of `ProcessBankData` floating around, but it is a feature I've wanted to add in for a long time.